karabo_data/components.py

```diff
@@ -82,7 +82,9 @@ class MPxDetectorBase:
     """
 
     _source_re = re.compile(r'(?P<detname>.+)/DET/(\d+)CH')
-    module_shape = (0, 0)  # Override in subclass
+    n_modules = 16
```

Comment:
Shouldn't that be part of the individual detector classes? I mean there might be the case when a detector does not consist of 16 modules?

Reply:
I was thinking that it could be overridden by detectors with other numbers of modules, but as the 3 we have so far are all composed of 16 modules, it's an easy shortcut to have 16 as the default.

What do you reckon, should we set it on each of the subclasses to be more explicit?

Reply:
Yes that's a fair point. Let's keep it then. Other than that LGTM